src/Utils.php

```diff
@@ -176,6 +172,28 @@ public static function getFirstValue(array $values, $returnKey = false)
         }
     }
 
+
+    /**
+     * Returns values as array
+     *
+     * @param array $values    The array provided by self::getData()
+     * @param bool  $returnKey Whether or not return the key instead the value
+     *
+     * @return array
+     */
+    public static function getArrayValue(array $values, $returnKey = false)
```

Comment:
Maybe, a better name for this function is `getAllValues`, more understandable, what do you think?